priv/elevators/elevator_station_data.csv

```diff
@@ -0,0 +1,227 @@
+test,elevator_id,line,parent_station,Station (for Betsy),children_ids_served,elevator_type,escalator_direction,facility_name,alternate_elevator_ids,alternate_service_long_text,Entering System Categorization,Exiting System Categorization,What station to EXIT at,Short Text,Questions/Notes,Is direction important?,Is the line/mode important?,secondary_alternate_service_text,elevator_name,alternate_service_short_text,facility_lat,facility_lon

```

Comment:
It doesn't seem like we need to commit this file; it's only read by the script, which produces another file which is then committed. The instructions could say to download the CSV to a temporary directory, outside the git repo.

Reply:
I was going back and forth on this. You're right, I'll take it out of the commit. Can also add the path as a script param.